sources/packages/backend/apps/api/src/route-controllers/education-program/_tests_/e2e/education-program.aest.controller.getProgramsSummary.e2e-spec.ts

```diff
@@ -0,0 +1,775 @@
+import { HttpStatus, INestApplication } from "@nestjs/common";
+import {
+  E2EDataSources,
+  createE2EDataSources,
+  createFakeInstitution,
+  createFakeInstitutionLocation,
+  createFakeUser,
+} from "@sims/test-utils";
+import {
+  AESTGroups,
+  BEARER_AUTH_TYPE,
+  createFakeEducationProgram,
+  createTestingAppModule,
+  getAESTToken,
+} from "../../../../testHelpers";
+import {
+  Institution,
+  InstitutionLocation,
+  ProgramStatus,
+  User,
+} from "@sims/sims-db";
+import * as request from "supertest";
+import {
+  appendRequiredString,
+  removeUnwantedString,
+} from "@sims/test-utils/utils/string-utils";
+
+describe("EducationProgramAESTController(e2e)-getProgramsSummary", () => {
+  let app: INestApplication;
+  let db: E2EDataSources;
+  let institution: Institution;
+  let institutionLocation: InstitutionLocation;
+  let sharedUser: User;
+
+  beforeAll(async () => {
+    const { nestApplication, dataSource } = await createTestingAppModule();
+    app = nestApplication;
+    db = createE2EDataSources(dataSource);
+    sharedUser = createFakeUser();
+    await db.user.save(sharedUser);
+  });
+
+  beforeEach(async () => {
+    institution = createFakeInstitution();
+    institutionLocation = createFakeInstitutionLocation({
+      institution: institution,
+    });
+    await db.institutionLocation.save(institutionLocation);
+  });
+
+  it(
+    "Should not retrieve any education programs when the search for the programs is made and no program statuses are searched for " +
```

Comment:
Please leave the white space at the end or at the beginning of the string. As it is it will have two white spaces. If possible, leave it at the end of every required string.

Reply:
Fixed it in all the tests. Leaving white space at the end everywhere.